packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -83,9 +83,23 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
       orderDrugs(items, patientUuid, abortController).then((erroredItems) => {
         setItems(erroredItems);
         fetchActivePatientOrders();
-
         if (erroredItems.length == 0) {
+          showToast({
+            kind: 'success',
+            title: 'Order placed',
```

Comment:
Can you also use add translations here (using `t`) for the `title` and below for the `description`? It's currently quite inconsistent with 2 strings being translated and 2 strings being hardcoded.